embedded-wallets/sdk/client-side/javascript.mdx

```diff
@@ -1312,20 +1312,20 @@ Starts the OAuth 2.0 login flow.
 
 **Valid Providers**
 
-|           |               |                                                          |
-| --------- | ------------- | -------------------------------------------------------- |
-| Name      | Argument      | Example                                                  |
-| Google    | `'google'`    | [Google demo](https://magic-oauth-v2-dun.vercel.app)    |
```

Comment:
Not totally averse to it, and I'm sure our new demo will eventually replace it, but what's the reasoning on removing this oauth v2 demo from all pages entirely? Maybe adding it to the OAuth implementation page would be good?